config.yml

```diff
@@ -9598,8 +9598,8 @@ production_recipes:
         type: LOOT_BONUS_BLOCKS
         level: 2
         probability: 0.36
-  Create_Wooden_Axe:
-    name: Create Wooden_Axe
+  Create_Wooden_Pickaxe:
+    name: Create Wooden Pickaxe
     production_time: 40
     inputs:
       Wooden Axe:
```

Comment:
I missed these -- the "label" on the input needs to be Wooden Pickaxe as well, to prevent confusion.